src/UnityPackage/Assets/Plugins/Web3/Scripts/ProjectConfigScriptableObject.cs

```diff
@@ -35,9 +37,11 @@ public string Network
         set => network = value;
     }
 
-    public string RPC
+    public string Rpc
     {
         get => rpc;
         set => rpc = value;
     }
+
+    public static ProjectConfigScriptableObject LoadDefault() => Resources.Load<ProjectConfigScriptableObject>(DefaultAssetPath);
```

Comment:
I have an entire static class dedicated to loading and saving the project config in the UPM PR. Let's put this PR on top of that and use that class.